java/java.openjdk.project/src/org/netbeans/modules/java/openjdk/project/customizer/BuildCategory.java

```diff
@@ -0,0 +1,132 @@
+                .addContainerGap()
+                .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+                    .addGroup(layout.createSequentialGroup()
+                        .addGap(21, 21, 21)
+                        .addComponent(antBuildLocationLabel)
+                        .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.RELATED)
+                        .addComponent(antBuildLocation))
+                    .addComponent(useAntBuild, javax.swing.GroupLayout.DEFAULT_SIZE, 380, Short.MAX_VALUE))
+                .addContainerGap())
+        );
+        layout.setVerticalGroup(
+            layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
+            .addGroup(layout.createSequentialGroup()
+                .addContainerGap()
+                .addComponent(useAntBuild)
+                .addPreferredGap(javax.swing.LayoutStyle.ComponentPlacement.UNRELATED)
+                .addGroup(layout.createParallelGroup(javax.swing.GroupLayout.Alignment.BASELINE)
+                    .addComponent(antBuildLocationLabel)
+                    .addComponent(antBuildLocation, javax.swing.GroupLayout.PREFERRED_SIZE, javax.swing.GroupLayout.DEFAULT_SIZE, javax.swing.GroupLayout.PREFERRED_SIZE))
+                .addContainerGap(244, Short.MAX_VALUE))
+        );
+    }// </editor-fold>//GEN-END:initComponents
+
+    private void useAntBuildActionPerformed(java.awt.event.ActionEvent evt) {//GEN-FIRST:event_useAntBuildActionPerformed
+        updateEnableDisable();
+    }//GEN-LAST:event_useAntBuildActionPerformed
+
+
+    // Variables declaration - do not modify//GEN-BEGIN:variables
+    private javax.swing.JTextField antBuildLocation;
+    private javax.swing.JLabel antBuildLocationLabel;
+    private javax.swing.JCheckBox useAntBuild;
+    // End of variables declaration//GEN-END:variables
+
+    @ProjectCustomizer.CompositeCategoryProvider.Registration(projectType=JDKProject.PROJECT_KEY, position=100)
+    public static ProjectCustomizer.CompositeCategoryProvider createCategoryProvider() {
+        return new ProjectCustomizer.CompositeCategoryProvider() {
+            @Override
+            public ProjectCustomizer.Category createCategory(Lookup context) {
+                if (context.lookup(Settings.class) != null)
+                    return ProjectCustomizer.Category.create("build", "Build", null);
+                return null;
+            }
+            @Override
+            public JComponent createComponent(ProjectCustomizer.Category category, Lookup context) {
+                Settings settings = context.lookup(Settings.class);
+                BuildCategory panel = new BuildCategory(settings.isUseAntBuild(), settings.getAntBuildLocation());
+                category.setOkButtonListener(evt -> {
+                    settings.setUseAntBuild(panel.useAntBuild.isSelected());
+                    settings.setJTregLocation(panel.antBuildLocation.getText());
```

Comment:
Should use setAntBuildLocation.
```suggestion
                    settings.setAntBuildLocation(panel.antBuildLocation.getText());
```